bucket/smartsystemmenu.json

```diff
@@ -0,0 +1,19 @@
+{
+    "version": "2.19.0",
+    "description": "A tool extends system menu of all windows in the system.",
+    "homepage": "https://github.com/AlexanderPro/SmartSystemMenu",
+    "license": "MIT",
+    "url": "https://github.com/AlexanderPro/SmartSystemMenu/releases/download/v2.19.0/SmartSystemMenu_v2.19.0.zip",
+    "hash": "c83980cfe4a011d9defb42ecbc3e54249f6186293735e369dcf10e5fb154de21",
+    "shortcuts": [
+        [
+            "SmartSystemMenu.exe",
+            "SmartSystemMenu"
+        ]
+    ],
+    "bin": "SmartSystemMenu.exe",
```

Comment:
Commandline arguments?

Reply:
Yes, it has cli, https://github.com/AlexanderPro/SmartSystemMenu#command-line-interface.

Reply:
> Yes, it has cli, https://github.com/AlexanderPro/SmartSystemMenu#command-line-interface.

Thanks for explaining. I didn't realize I had to explain XD